README.md

```diff
@@ -22,7 +22,8 @@ https://codesoapbox.dev/keycloak-user-migration
 
 | Keycloak Version | Version/Commit                                                                                                                                     |
 |------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
-| 24.X             | [4.0.0](https://github.com/daniel-frak/keycloak-user-migration/releases/tag/4.0.0)                                                                                                                                           |
+| 25.X             | [5.0.0](https://github.com/daniel-frak/keycloak-user-migration/releases/tag/5.0.0)                                                                 |
```

Comment:
This version should just be be `SNAPSHOT` (with no links). The release workflow will change it to the correct tag automatically and that way we won't have dead links.